metaflow/cli.py

```diff
@@ -618,10 +611,10 @@ def run(obj,
     write_run_id(run_id_file, runtime.run_id)
 
     parameters.set_parameters(obj.flow, kwargs)
+
```

Comment:
why the new line? Let's remove because we aren't changing this code

Reply:
Done